app/grandchallenge/cases/serializers.py

```diff
@@ -53,3 +68,23 @@ class Meta:
             "algorithm_image",
             "api_url",
         ]
+
+
+class RawImageFileSerializer(serializers.ModelSerializer):
+    upload_session = HyperlinkedRelatedField(
+        queryset=RawImageUploadSession.objects.all(),
+        view_name="api:upload-session-detail",
+    )
+
+    def validate(self, attrs):
+        upload_session = attrs["upload_session"]
+        user = self.context.get("request").user
+        if not user.has_perm("view_rawimageuploadsession", upload_session):
```

Comment:
This modifies the raw image upload session, so we want to check for `change_rawimageuploadsession`.